package.json

```diff
@@ -20,17 +20,54 @@
     "dev": "NODE_ENV=development webpack --progress",
     "watch": "NODE_ENV=development webpack --watch --progress",
     "css": "WEBPACK_CHILD=css npm run build",
+    "test": "jest",
+    "coverage": "jest --coverage",
     "lock": "npm-shrinkwrap --dev",
     "lint": "eslint client/src && sass-lint client/src",
     "lint-js": "eslint client/src",
     "lint-js-fix": "eslint client/src --fix",
     "lint-sass": "sass-lint client/src"
   },
+  "jest": {
+    "roots": [
+      "client/src"
+    ],
+    "modulePaths": [
+      "client/src",
+      "../admin/client/src",
+      "../admin/node_modules",
+      "vendor/silverstripe/admin/client/src",
+      "vendor/silverstripe/admin/node_modules"
+    ],
+    "testMatch": [
+      "**/tests/**/*-test.js?(x)"
+    ],
+    "transform": {
+      ".*": "babel-jest"
+    }
+  },
+  "babel": {
+    "presets": [
+      "env",
+      "react"
+    ],
+    "plugins": [
+      "transform-object-rest-spread"
+    ]
+  },
   "devDependencies": {
     "@silverstripe/eslint-config": "^0.0.4",
     "@silverstripe/webpack-config": "^0.8.0",
-    "expose-loader": "^0.7.4",
-    "sass-loader": "^6.0.7"
+    "@storybook/addon-actions": "^3.2.19",
+    "@storybook/addon-options": "^3.2.19",
+    "@storybook/react": "^3.2.19",
```

Comment:
Were you going to add a story book definition for it? That'd be cool, but not required

Reply:
I was playing around with that but I didn't get around to it. I can remove that.